mlflow/sklearn/__init__.py

```diff
@@ -945,6 +1059,50 @@ def patched_fit(original, self, *args, **kwargs):
                     FLAVOR_NAME, class_def, func_name, patched_fit, manage_run=True,
                 )
 
+        for func_name in ["predict", "transform", "predict_proba"]:
+            if hasattr(class_def, func_name):
+                safe_patch(
+                    FLAVOR_NAME, class_def, func_name, patched_predict, manage_run=False,
+                )
+
+    from sklearn import metrics
+    # Note:
+    #  don't need patch model.score() because internally model.score() will call the concrete metric API
```

Comment:
Is this really true?

https://github.com/scikit-learn/scikit-learn/blob/82df48934eba1df9a1ed3be98aaace8eada59e6e/sklearn/covariance/_empirical_covariance.py#L220

Reply:
ok if there's such case then we can patch model.score(), and when code run into `model.score()` scope we can unpatch the metric API temporarily.